### ISSUE TITLE
Tab order is not following fields order

### ISSUE THREAD

**Issue by gotson** [mstarke/MacPass#98]
When pressing tab, the focus goes to random elements instead of the next logical one.
I know it's an alpha release, and fields will most likely change until the first release, but at least now it's mentioned somewhere :)

**Comment by mstarke** [comment]
Tab order in entry/group inspector works with the exception of custom fields as the table-view does not tab-out after the last entry is selected. This should be a separate issue
